sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -74,6 +74,40 @@ double joint_matmul(TOperand *A, TOperand *B, TResult *C, queue &q, int i) {
           auto m1 = it.get_local_id(0);
           auto n1 = it.get_local_id(1) / sgSize;
           auto sg = it.get_sub_group();
+#ifdef PREFETCH
+          size_t sgId = sg.get_group_id()[0];
+#ifdef VNNI
+          // each warp gets 8x32. But 16 warps are needed for each row (512)
```

Comment:
what is 512 and where 16 is coming from?

Reply:
Sure, I will add more explanation